modules/administration_manual/pages/configuration/user/user_auth_ftp_smb_imap.adoc

```diff
@@ -23,9 +23,9 @@ Currently the https://github.com/owncloud/apps[External user support
 app] (user_external), _which is not enabled by default_, provides three
 backends. These are:
 
-* link:imap[IMAP]
-* link:smb[SMB]
-* link:ftp[FTP]
+* xref:imap[IMAP]
```

Comment:
This don't appear valid, as xref's need to either end with `.adoc` or `#name-of-a-valid-anchor`.